src/aap_eda/settings/default.py

```diff
@@ -696,3 +697,20 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 API_PATH_TO_UI_PATH_MAP = settings.get(
     "API_PATH_UI_PATH_MAP", {"/api/controller": "/execution", "/": "/#"}
 )
+
+
+ANSIBLE_LOCAL_TEMP = settings.get("ANSIBLE_LOCAL_TEMP", tempfile.gettempdir())
+if not os.access(ANSIBLE_LOCAL_TEMP, os.W_OK):
+    raise ImproperlyConfigured(
+        f"ANSIBLE_LOCAL_TEMP {ANSIBLE_LOCAL_TEMP} is not writable, "
+        "please define ANSIBLE_LOCAL_TEMP with a writable directory."
+    )
+
+ANSIBLE_REMOTE_TEMP = settings.get(
```

Comment:
I'm concerned to use the same default value for ANSIBLE_REMOTE_TEMP and ANSIBLE_LOCAL_TEMP just in case there might be conflicts, have different purposes. 

I don't see ANSIBLE_REMOTE_TEMP in https://docs.ansible.com/ansible/latest/reference_appendices/config.html
Seems that is [going to be deprecated](https://github.com/ansible/ansible/blob/stable-2.5/changelogs/CHANGELOG-v2.5.rst#minor-changes-4)

I think it is better to define the variable in the inventory instead of an envvar and use a tempdir created on demand. Thoughts?

Reply:
Actual tests proved that we need to set ANSIBLE_REMOTE_TEMP.

Since we're already a creating temp folder for project sync, we can just create new temp folders for ANSIBLE too. This will eliminate the needs to set temp folder for ANSIBLE in the settings.

Reply:
I'm not saying you don't have to set the remote temp, I'm proposing to define the variable in the inventory/playbook, but in anycase it should be better ANSIBLE_REMOTE_TMP, because ANSIBLE_REMOTE_TEMP [will be deprecated](https://github.com/ansible/ansible/blob/stable-2.5/changelogs/CHANGELOG-v2.5.rst#minor-changes-4#:~:text=ANSIBLE_REMOTE_TMP).  It would be a more robust solution in the long term. 
Plus a temp dir instead of a static setting.

Ref: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/sh_shell.html#parameter-remote_tmp